Update addons-test-utils.md (#3660)
reactjs/reactjs.org: content/docs/addons-test-utils.md

> Note:
>
> We recommend using [React Testing Library](https://testing-library.com/react) which is designed to enable and encourage writing tests that use your components as the end users do.
> 

> > For React versions <= 16, a testing utility called [Enzyme](https://airbnb.io/enzyme/) makes it easy to assert, manipulate, and traverse your React Components' output.

Reviewer: ```suggestion
> For React versions <= 16, the [Enzyme](https://airbnb.io/enzyme/) library makes it easy to assert, manipulate, and traverse your React Components' output.
```
